ui/packages/consul-ui/app/templates/dc/acls/auth-methods/show/binding-rules.hbs

```diff
@@ -0,0 +1,28 @@
+
+<div class="tab-section">
+{{#if (gt bindingRules.length 0)}}
+  <p>Binding rules allow an operator to express a systematic way of automatically linking roles and service identities to newly created tokens without operator intervention.
+  </p>
+  <p>Successful authentication with an auth method returns a set of trusted identity attributes corresponding to the authenticated identity. Those attributes are matched against all configured binding rules for that auth method to determine what privileges to grant the Consul ACL token it will ultimately create.
+  </p>
```

Comment:
Hey I just came back here as I noticed in your namespace rules PR that I think we may need to put these p tags outside of the conditional, as I said in the other PR, maybe check in with design first as I'm not 100% sure